src/managementgroup/azext_managementgroup/generated/_params.py

```diff
@@ -0,0 +1,166 @@
+                   'element will include a token parameter that specifies a starting point to use for subsequent '
+                   'calls.')
+        c.argument('skip', type=int, help='Number of entities to skip over when retrieving results. Passing this in '
+                   'will override $skipToken.')
+        c.argument('top', type=int, help='Number of elements to return when retrieving results. Passing this in will '
+                   'override $skipToken.')
+        c.argument('select', type=str, help='This parameter specifies the fields to include in the response. Can '
+                   'include any combination of Name,DisplayName,Type,ParentDisplayNameChain,ParentChain, e.g. '
+                   '\'$select=Name,DisplayName,Type,ParentDisplayNameChain,ParentNameChain\'. When specified the '
+                   '$select parameter can override select in $skipToken.')
+        c.argument('search', arg_type=get_enum_type(['AllowedParents', 'AllowedChildren',
+                                                     'ParentAndFirstLevelChildren', 'ParentOnly', 'ChildrenOnly']),
+                   help='The $search parameter is used in conjunction with the $filter parameter to return three '
+                   'different outputs depending on the parameter passed in.  With $search=AllowedParents the API will '
+                   'return the entity info of all groups that the requested entity will be able to reparent to as '
+                   'determined by the user\'s permissions. With $search=AllowedChildren the API will return the entity '
+                   'info of all entities that can be added as children of the requested entity. With '
+                   '$search=ParentAndFirstLevelChildren the API will return the parent and  first level of children '
+                   'that the user has either direct access to or indirect access via one of their descendants. With '
+                   '$search=ParentOnly the API will return only the group if the user has access to at least one of '
+                   'the descendants of the group. With $search=ChildrenOnly the API will return only the first level '
+                   'of children of the group entity info specified in $filter.  The user must have direct access to '
+                   'the children entities or one of it\'s descendants for it to show up in the results.')
+        c.argument('filter_', options_list=['--filter'], type=str, help='The filter parameter allows you to filter on '
+                   'the the name or display name fields. You can check for equality on the name field (e.g. name eq '
+                   '\'{entityName}\')  and you can check for substrings on either the name or display name fields(e.g. '
+                   'contains(name, \'{substringToSearch}\'), contains(displayName, \'{substringToSearch\')). Note that '
+                   'the \'{entityName}\' and \'{substringToSearch}\' fields are checked case insensitively.')
```

Comment:
Please refine the help message, because It seems to illustrate the feature of this parameter from the perspective of REST API rather than from the perspective of command parameters.
Please explain the parameters from the perspective of command parameters, otherwise users will be confused.